maven.indexer/external/indexer-core-6.0.0-notice.txt

```diff
@@ -0,0 +1,5 @@
+Maven :: Indexer Core
+Copyright 2002-2017 The Apache Software Foundation
+
+This product includes software developed at
```

Comment:
I believe we can remove these two lines?

Reply:
We could, but we don't need to do - the ant task, that merges the notice files, already removes the "This product includes ..." lines. So I took the whole notice file.

Reply:
OK.